src/main/java/dao/Dao.java

```diff
@@ -1,13 +1,15 @@
 package dao;
+import exception.DrugNotFoundException;
+
 import java.util.List;
 
 
 public interface Dao<T> {
-    T get(int id) throws CloneNotSupportedException;
+    T findById(int id) throws CloneNotSupportedException;
```

Comment:
Why user should care about handling some CloneNotSupportedException when finding an object?